doc/authors.md

```diff
@@ -2,9 +2,9 @@ Zcash Contributors
 ==================
 
 Jack Grigg (1335)
-Kris Nuttycombe (685)
+Kris Nuttycombe (698)
 Simon Liu (460)
-Sean Bowe (403)
+Sean Bowe (404)
 Daira Hopwood (396)
```

Comment:
Daira Hopwood and Daira Emma Hopwood should be merged, using the latter name. I will open a PR to do that when I get back.